pkgs/stdenv/freebsd/make-bootstrap-tools.nix

```diff
@@ -0,0 +1,102 @@
+{ pkgs ? import ../../.. {} }:
+  let
+    inherit (pkgs) runCommand lib;
+    # splicing doesn't seem to work right here
+    inherit (pkgs.buildPackages) nix rsync;
+    pack-all =
+      packCmd: name: pkgs: fixups:
+      (runCommand name {
+        requiredSystemFeatures = [ "recursive-nix" ];
```

Comment:
same comment as before. this seems to require enabling `recursive-nix` in the daemon. is it enabled on hydra?

Reply:
This is a question for someone who actually understands the hydra configuration or knows how to reverse engineer it!

Reply:
We can figure this out later as of now bootstrap tarballs are downloaded from localhost. It shouldn't be a big problem to get this enabled.

Reply:
I have some ideas that might help

1. There is the weird full closure thing that we use for the installation media IIRC
2. Just use static binaries for bootstrap stuff, and do not have libraries or references at all

Reply:
2 is assuredly not going to work since recursive-nix is required for not just `unpack.nar.xz` but also `bootstrap-tools.tar.xz`. I'd be interesting in doing some hacking on 1 if I could be given some pointers of where to start, but my favorite solution is still just enable recursive-nix in hydra.

Reply:
I gotta download that NixBSD VM and get hacking :)